pgxn/remotexact/rwset.h

```diff
@@ -51,10 +51,11 @@ typedef struct RWSetTuple
  */
 typedef struct RWSetRelation
 {
+	bool		is_index;
```

Comment:
Why? Doesn't it make sense to group the bools together?

Reply:
the first thing we always do when reading from a relation is check whether it is an index or not, so I thought this made reading the code a bit easier

Reply:
Also this makes the field order consistent with the serialization order.

Reply:
Makes sense.